testing/upload_validation.ts

```diff
@@ -1194,43 +1211,52 @@ const legacyPackMetadataSchema = validateFormulas(
         "Specify the domain that your pack makes http requests to using `networkDomains: ['example.com']`",
       path: ['networkDomains'],
     },
-  )
-  .superRefine((untypedData, context) => {
+  );
+const packMetadataSchemaBySdkVersion = [
+  {
     // Check that packs with multiple network domains explicitly choose which domain gets auth.
+    // This is a backward-incompatible validation that takes effect in any pack release after 0.9.0.
+    versionRange: '>0.9.0',
```

Comment:
is this >0.9.0 or >=0.9.0? just double check. I don't know which versions they were added.

Reply:
The code was changed while the SDK version was still 0.9.0 without bumping the SDK version, so I used `>0.9.0` to say, "whatever the next version is"

Turns out the next release was `0.10.0`, so there would only be a `0.9.1` release if we decided we wanted to maintain the `0.9` branch do a bugfix or security patch release